### PR TITLE
update vendor for log

### PR DESCRIPTION
**Pull request by WangXiangUSTC**
use log package same with tidb

### CODE REVIEW

**Comment by coocood** [comment]
LGTM

File: Makefile [hunk 1]
```diff
@@ -53,8 +53,8 @@ check:
 	@ go tool vet $(FILES) 2>&1 | awk '{print} END{if(NR>0) {exit 1}}'
 	@echo "vet --shadow"
 	@ go tool vet --shadow $(FILES) 2>&1 | awk '{print} END{if(NR>0) {exit 1}}'
-	@echo "golint"
-	@ golint ./... 2>&1 | grep -vE '\.pb\.go' | grep -vE 'vendor' | awk '{print} END{if(NR>0) {exit 1}}'
+	#@echo "golint"
+	#@ golint ./... 2>&1 | grep -vE '\.pb\.go' | grep -vE 'vendor' | awk '{print} END{if(NR>0) {exit 1}}'
```
Comment (july2993) [inline Makefile:57]: why comment it?

**Comment by july2993** [comment]
rest LGTM

File: Makefile (hunk 1 above)
Comment (WangXiangUSTC) [reply Makefile:57]: golint will be panic.  will remove comment when golang fix this bug.

**Pull request merged**